neurobooth_os/tasks/saccade/saccade_task.py

```diff
@@ -160,5 +160,7 @@ def run_trials(self, prompt=True):
 
 
 if __name__ == "__main__":
+    from neurobooth_os import config
```

Comment:
is this the code that is designed to make this easier to test?  
In general I'm not a fan of test code in production code.  I know that's standard practice in python code, but it's not generally accepted in code for production applications.  I would make the same comment in the Smooth Pursuit code.